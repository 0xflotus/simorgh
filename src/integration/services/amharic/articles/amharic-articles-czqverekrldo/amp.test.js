/**
 * WARNING: DO NOT EDIT. This file is automatically generated by src/integration/utils/runTests/writeTestFiles/writeTestFile.js
 * @service amharic
 * @pathname /amharic/articles/czqverekrldo
 */

import runAmpTests from '../../../../pages/articles/ampTests';

describe('AMP amharic articles', runAmpTests);
