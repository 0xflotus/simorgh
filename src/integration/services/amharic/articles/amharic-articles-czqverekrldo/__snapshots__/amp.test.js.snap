// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AMP amharic articles A11y I can see the skip to content link 1`] = `"ወደ ዋናው ይዘት ይለፉ"`;

exports[`AMP amharic articles AMP Consent config 1`] = `
Object {
  "consents": Object {
    "user-consent": Object {
      "promptIfUnknownForGeoGroup": "eea",
      "promptUI": "consent-prompt",
    },
  },
  "policy": Object {
    "default": Object {
      "timeout": Object {
        "fallbackAction": "reject",
        "seconds": 0,
      },
      "waitFor": Object {
        "user-consent": Array [],
      },
    },
  },
}
`;

exports[`AMP amharic articles AMP Geo config 1`] = `
Object {
  "ISOCountryGroups": Object {
    "eea": Array [
      "at",
      "ax",
      "be",
      "bg",
      "bl",
      "cy",
      "cz",
      "de",
      "dk",
      "ea",
      "ee",
      "es",
      "fi",
      "fr",
      "gb",
      "gf",
      "gg",
      "gi",
      "gp",
      "gr",
      "hr",
      "hu",
      "ic",
      "ie",
      "im",
      "is",
      "it",
      "je",
      "li",
      "lt",
      "lu",
      "lv",
      "mf",
      "mq",
      "mt",
      "nc",
      "nl",
      "no",
      "pf",
      "pl",
      "pm",
      "pt",
      "re",
      "ro",
      "se",
      "si",
      "sj",
      "sk",
      "tf",
      "va",
      "wf",
      "yt",
    ],
  },
}
`;

exports[`AMP amharic articles Analytics ATI 1`] = `
Object {
  "requests": Object {
    "base": "https://logws1363.ati-host.net?",
    "pageview": "\${base}s=598343&s2=4&p=amharic.articles.czqverekrldo.page&r=\${screenWidth}x\${screenHeight}x\${screenColorDepth}&re=\${availableScreenWidth}x\${availableScreenHeight}&hl=\${timestamp}&lng=\${browserLanguage}&x1=[urn:bbc:optimo:czqverekrldo]&x2=[amp]&x3=[news-amharic]&x4=[am]&x5=[\${sourceUrl}]&x6=[\${documentReferrer}]&x7=[article]&x8=[simorgh]&x9=[የዚህ+የሙከራ+ጽሑፍ+ርዕስ+ይህ+ነው]&x11=[2019-11-27T11:32:54.815Z]&x12=[2019-11-27T11:32:54.815Z]&ref=\${documentReferrer}",
  },
  "transport": Object {
    "beacon": false,
    "image": true,
    "xhrpost": false,
  },
  "triggers": Object {
    "trackPageview": Object {
      "on": "visible",
      "request": "pageview",
    },
  },
}
`;

exports[`AMP amharic articles Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/amharic"`;

exports[`AMP amharic articles Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC. ቢቢሲ ከሌሎች ድረ-ገጾች ለሚመጡ መረጃዎች ሀላፊነት አይወስድም. ስለ ውጪ ሊንኮች ያለን አቀራረብ"`;

exports[`AMP amharic articles Header I can see the branding 1`] = `"BBC News, አማርኛ"`;

exports[`AMP amharic articles I can see an image with a caption 1`] = `"https://ichef.bbci.co.uk/news/640/cpsdevpb/6ed8/test/cfebd050-1108-11ea-acaa-cd607a38aea4.jpg"`;

exports[`AMP amharic articles I can see an image with a caption 2`] = `"የምስሉ መግለጫ, ይህ የሙከራ ምስል የቅጂ መብት ቢቢሲ የፈረንሳይን ካርታ ያሳያል ፡፡ ምስሉ በመጀመሪያዎቹ ሦስት ብሎኮች ውስጥ የሚገኝ ሲሆን ይህ መግለጫ ጽሑፍ አለው ፡፡"`;

exports[`AMP amharic articles SEO Canonical link 1`] = `"http://localhost:7080/amharic/articles/czqverekrldo"`;

exports[`AMP amharic articles SEO Dir attribute 1`] = `"ltr"`;

exports[`AMP amharic articles SEO FB admins 1`] = `"100004154058350"`;

exports[`AMP amharic articles SEO FB app ID 1`] = `"1609039196070050"`;

exports[`AMP amharic articles SEO Heading level 1 1`] = `"የዚህ የሙከራ ጽሑፍ ርዕስ ይህ ነው"`;

exports[`AMP amharic articles SEO Lang attribute 1`] = `"am"`;

exports[`AMP amharic articles SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "Article",
  "author": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/amharic.png",
      "width": 1024,
    },
    "name": "BBC News አማርኛ",
    "noBylinesPolicy": "https://www.bbc.com/amharic/institutional-49283133#authorexpertise",
  },
  "dateModified": "2019-11-27T11:32:54.815Z",
  "datePublished": "2019-11-27T11:32:54.815Z",
  "headline": "የዚህ የሙከራ ጽሑፍ ርዕስ ይህ ነው",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/amharic.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/amharic/articles/czqverekrldo",
    "@type": "WebPage",
    "name": "የዚህ የሙከራ ጽሑፍ ርዕስ ይህ ነው",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/amharic.png",
      "width": 1024,
    },
    "name": "BBC News አማርኛ",
    "publishingPrinciples": "https://www.bbc.com/amharic/institutional-49283133",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/amharic.png",
  "url": "http://localhost:7080/amharic/articles/czqverekrldo",
}
`;

exports[`AMP amharic articles SEO OG description 1`] = `"የዚህ የሙከራ ጽሑፍ ርዕስ ይህ ነው"`;

exports[`AMP amharic articles SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/amharic.png"`;

exports[`AMP amharic articles SEO OG image alt 1`] = `"BBC News አማርኛ"`;

exports[`AMP amharic articles SEO OG locale 1`] = `"am-ET"`;

exports[`AMP amharic articles SEO OG site name 1`] = `"BBC News አማርኛ"`;

exports[`AMP amharic articles SEO OG title 1`] = `"የዚህ የሙከራ ጽሑፍ ርዕስ ይህ ነው - BBC News አማርኛ"`;

exports[`AMP amharic articles SEO OG type 1`] = `"article"`;

exports[`AMP amharic articles SEO OG url 1`] = `"http://localhost:7080/amharic/articles/czqverekrldo"`;

exports[`AMP amharic articles SEO Page title 1`] = `"የዚህ የሙከራ ጽሑፍ ርዕስ ይህ ነው - BBC News አማርኛ"`;

exports[`AMP amharic articles SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`AMP amharic articles SEO Twitter card 1`] = `"summary_large_image"`;

exports[`AMP amharic articles SEO Twitter creator 1`] = `"@bbcnews"`;

exports[`AMP amharic articles SEO Twitter description 1`] = `"የዚህ የሙከራ ጽሑፍ ርዕስ ይህ ነው"`;

exports[`AMP amharic articles SEO Twitter image alt 1`] = `"BBC News አማርኛ"`;

exports[`AMP amharic articles SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/amharic.png"`;

exports[`AMP amharic articles SEO Twitter site 1`] = `"@bbcnews"`;

exports[`AMP amharic articles SEO Twitter title 1`] = `"የዚህ የሙከራ ጽሑፍ ርዕስ ይህ ነው - BBC News አማርኛ"`;
